setup.py

```diff
@@ -3,21 +3,21 @@
 # from https://stackoverflow.com/a/9079062
 import sys  
 if sys.version_info[0] < 3:
-    raise Exception("The fenicsadapter only supports Python3. Did you run $python setup.py <option>.? Try running $python3 setup.py <option>.")
+    raise Exception("The fenicsprecice only supports Python3. Did you run $python setup.py <option>.? Try running $python3 setup.py <option>.")
 
 # using point 3. in https://packaging.python.org/guides/single-sourcing-package-version/#single-sourcing-the-version
 version = {}
-with open("fenicsadapter/version.py") as fp:
+with open("fenicsprecice/version.py") as fp:
     exec(fp.read(), version)
 
-setup(name='fenicsadapter',
+setup(name='fenicsprecice',
       version=version['__version__'],
-      description='preCICE-adapter for the open source computing platform FEniCS ',
+      description='FEniCS-preCICE adapter: A preCICE-adapter for the open source computing platform FEniCS.',
```

Comment:
```suggestion
      description='FEniCS-preCICE adapter is a preCICE adapter for the open source computing platform FEniCS.',
```
I checked what numpy is doing here:
```
$ pip3 show numpy
Name: numpy
Version: 1.17.4
Summary: NumPy is the fundamental package for array computing with Python.
Home-page: https://www.numpy.org
...
```
They use the name of the software (including "complicated" typography) in the `description` (Summary). Applied to our software this means we should use the official name "FEniCS-preCICE adapter" here:
```
$ pip3 show fenicsprecice
Name: fenicsprecice
Version: 0.2
Summary: FEniCS-preCICE adapter is a preCICE adapter for the open source computing platform FEniCS.
Home-page: https://github.com/precice/fenics-adapter
...
```
Does this look good?